### ISSUE TITLE
Project is using deprecated .expo.* file extensions

### ISSUE THREAD

**Issue by idolize** [benevbright/react-navigation-collapsible#202]
## Information
- react-native version: 0.64.2 (expo SDK 43)
- react-navigation version: "@react-navigation/native": "^6.0.6"
- react-navigation-collapsible version: 6.2.0
- Platform (iOS/Android): N/A
- react-native init or Expo: Expo

## Detail

Updated to [Expo SDK 43](https://blog.expo.dev/expo-sdk-43-aa9b3c7d5541) and see the following error while running `expo upgrade`:

```
✖ Found project files with deprecated features
Project is using deprecated .expo.* file extensions.
Please refactor the following files and upgrade modules accordingly:

- /Users/test/node_modules/react-navigation-collapsible/lib/src/platform/index.expo.js

Dangerously disable this check with EXPO_LEGACY_IMPORTS=1
Learn more: http://expo.fyi/expo-extension-migration
```

**Comment by benevbright** [comment]
fixed in 6.3.0. thanks again for reporting.

**Comment by idolize** [comment]
Thanks for the quick fix!
